### DISCUSSION TITLE
Simplify releasing WebGoat: just use one branch --> main

### DISCUSSION THREAD

**Discussion by nbaars** [WebGoat/WebGoat#1382] [General]
Since releasing takes some time with using `git-flow` with `develop` and `main` I think it is better to just use `main`.  The only time we use `main` is for releasing which is an overhead we can do without.

@aolle @zubcevic what do you think?

**Comment by zubcevic** [comment]
I am all for 1 main branch. Most of my projects now also use this approach. Especially if there are only a few contributors and more or less 1 active version to be maintained.

**Reply by zubcevic** [reply]
Since you made the release it makes sense to start using it straight away. Set the default branch to main and maybe even delete develop

**Comment by aolle** [comment]
Hi,
I agree. 
We also currently have some stale branches that maybe we can cleanup if desired.

**Comment by nbaars** [comment]
@zubcevic @aolle done, updated the default branch in Github. The develop branch is still there but it is locked.

**Comment by keiland-cylm** [comment]
Is anyone here rn? I downloaded webgoat and i dont have the server?
